graphql/user/updateSocialMedia.graphql

```diff
@@ -0,0 +1,17 @@
+mutation updateSocialMedia(
+  $facebook: String
+  $instagram: String
+  $youtube: String
+  $website: String
+  $linkedin: String
+) {
+  updateSocialMedia(
+    facebook: $facebook
+    instagram: $instagram
+    youtube: $youtube
+    website: $website
+    linkedin: $linkedin
+  ) {
+    url
+  }
+}
```

Comment:
I'm wondering if we could have simply used the existing `updateUser` mutation, since that's all already in place and `socialMedia` is a field on the `User` model? What was your thought behind creating a new, specific mutation for that field?